src/main/java/frc/robot/poseestimator/observations/VisionRobotPoseObservation.java

```diff
@@ -0,0 +1,11 @@
+package frc.robot.poseestimator.observations;
+
+import edu.wpi.first.math.geometry.Pose3d;
+
+public interface VisionRobotPoseObservation {
```

Comment:
RobotPoseVisionOservation, alsoi dont think robot pose is improtat in the name

Reply:
it is. What if would be an observation of note (or any object) location?

Reply:
It would return estimation and timestamp

Reply:
But that's not what it should do. 
btw, and it also include the apriltagheight for better integration with wpilib's poseestimator